src/reducers/accounts.js

```diff
@@ -147,6 +115,7 @@ export function deserializeAccounts(accounts: Account[]) {
     // as account can be passed by main process, the Date types
     // can be converted to string. we ensure here that we have real
     // date
+    // FIXME Account shouldn't be passed by main process. only AccountRaw should be!
```

Comment:
it shouldn't be the case anymore, so I guess we can get rid of that hack

Reply:
ok, i'll check later ;)